### ISSUE TITLE
Playlist.Once input causes invalid requests to be added to the queue

### ISSUE THREAD

**Issue by chrisnellis** [savonet/liquidsoap#1177]
**Describe the bug**
The `playlist.once` function continuously adds empty URLs to the queue after it finishes playing all songs in the playlist given.

**To Reproduce**
Here's an example script:
```
playOnce = playlist.once("once.pls")

mainSource = mksafe(playlist("mainSource.pls", mode="normal"))

outputCommand = fallback([playOnce, mainSource])

mountPointMp3 = "output"

output.icecast(%mp3,
  host="localhost",port=8000,password="hackme",
  mount=mountPointMp3,
  outputCommand)
```

As soon as the tracks in the `once.pls` file finish, there are many empty requests added to the queue. Here's the lines I get:
`[request:4] Nonexistent file or ill-formed URI ""!`
`[request:2] There are currently 100 RIDs, possible request leak! Please check that you don't have a loop on empty/unavailable requests, or creating requests without destroying them. Decreasing request.grace_time can also help.`

Here's a sample of `once.pls`:
```
[playlist]
File1=~/Music/track.mp3

NumberOfEntries=1
Version=2
```

**Expected behavior**
Such requests aren't made

**Version details**
 - OS: Mac OS 10.14.6
 - Liquidsoap Version 1.4.1

**Install method**
Installed `opam` through `brew` and installed `liquidsoap` through `opam` as described in the docs.

**Comment by toots** [comment]
Hi @chrisnellis thanks for reporting! We have a fix for that just ready to be shipped with our upcoming `1.4.2` bugfix release. If you want to test it:
```
git clone --recursive https://github.com/savonet/liquidsoap.git
cd liquidsoap && git checkout 1.4.2
opam install -y .
```

**Comment by chrisnellis** [comment]
That worked for me! Cheers!! 🍻
